.github/workflows/ci.yml

```
@@ -12,7 +12,8 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1' # automatically expands to the latest stable 1.x release of Julia
+          #- '1' # automatically expands to the latest stable 1.x release of Julia
+          #- '1.10'
```

If you want to test on 1.10.0-rc1:
```suggestion
          - '~1.10.0-0'
```

---

I mean, I hate that syntax, but I do want that CI more